saleor/graphql/payment/types.py

```diff
@@ -533,6 +537,10 @@ class Meta:
     def resolve_id(root: models.TransactionItem, _info: ResolveInfo):
         return root.token
 
+    @staticmethod
+    def resolve_token(root: models.TransactionItem, _info):
```

Comment:
I think it is not needed, as it should be handled by default by graphql engine 🤔